tools/build-bazel.cmd

```diff
@@ -0,0 +1,6 @@
+pushd "%~dp0"
+set "PATH=%CD%;%PATH%"
```

Comment:
Is setting path to CWD necessary?  Is this because you downloaded bazel to the project root?

Reply:
Yeah, that's not the root workspace but the `$project/tools` path. And indeed, that's actually where I'm downloading the `bazel.exe` into, so that I don't have it in my usual standard paths (or so that I can have different versions of it elsewhere in another build tree). We could change that, but it should be rather benign, as it allows you to prefer the `tools/` directory tools from your current local snapshot. Kinda additional isolation of tooling in a way.